run-tests-with-build-optawebs-parent/optawebs-project-sources-test/pom.xml

```diff
@@ -50,6 +59,9 @@
                 <groupId>org.apache.maven.plugins</groupId>
                 <artifactId>maven-invoker-plugin</artifactId>
                 <configuration>
+                    <properties>
+                        <maven.test.skip>true</maven.test.skip>
```

Comment:
this should not be in the general section, but rather the `openshift` profile, right? Otherwise we're skipping tests for default build (even when the profile is not active).
Not to have to copy the plugin config anywhere, it should be enough to create property `<maven.test.skip>` where default would be `false` and just `openshift` profile would set it to `true`.

Reply:
Yes, you right, fixed